tests/test_assign_to_workflow_sample_review.py

```diff
@@ -0,0 +1,42 @@
+from scripts.assign_sample_review import AssignWorkflowSampleReview
+from tests.test_common import TestEPP, fake_artifact
+from unittest.mock import Mock, patch, PropertyMock
+
+
+def fake_all_outputs(unique=False, resolve=False):
+    '''Return a list of mocked artifacts which contain sample which contain artifact ... Simple!'''
+    return (
+        Mock(samples=[Mock(artifact=fake_artifact('a1'), id='s1')], udf={"SR Useable": 'yes'}),
+        Mock(samples=[Mock(artifact=fake_artifact('a2'), id='s2')],udf={"SR Useable":'yes'})
```

Comment:
We could add an artifact marked as `'SR Useable': 'no'` to make sure that non-useable samples don't get routed to data release